package.json

```
@@ -3,12 +3,13 @@
   "version": "0.0.0",
   "private": true,
   "scripts": {
-    "start": "gulp dev:extension",
-    "dist": "gulp dist",
-    "start:test": "gulp dev:test",
+    "start": "yarn build dev",
+    "dist": "yarn build prod",
+    "build": "node development/build/index.js",
+    "start:test": "yarn build devTest",
```

Looks like you forgot to rename this one when you standardized all of the task names
```suggestion
    "start:test": "yarn build testDev",
```